CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing to cron <!-- omit in toc -->
+
+First off, thanks for taking the time to contribute! ❤️
+
+All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
+
+> And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
+
+> - Star the project
+> - Tweet about it
+> - Refer this project in your project's readme
+> - Mention the project at local meetups and tell your friends/colleagues
```

Comment:
I would add "join the Discord community" to this list. having people there to hear about announcements, ask and answer questions, etc. can also help the project

Reply:
I know you mention it below but probably a lot of people will only read one or two sections so I'd add it whenever relevant